src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &_minutes_hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    width: $minutes-width;
+    height: $minutes-height;
+    transform: $hands-transform;
+    background-color: #0700ff;
+    z-index: 1;
+    transform-origin: 50% 100%;
+    animation: rotate 3600s infinite steps(60);
+  }
+
+  &_seconds_hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    width: $seconds-width;
+    height: $seconds-height;
```

Comment:
The variable `$seconds-height` is not defined in the 'variables.scss' file. Please define it there before using it here.